.github/BOTMETA.yml

```diff
@@ -925,6 +925,7 @@ files:
     authors: Andersson007
     maintainers: Alexander198961 Xyon bmalynovytch bmildren kurtdavis michaelcoburn oneiroi tolland
     keywords: mariadb proxysql
+    ignore: tomaszkiewicz
```

Comment:
Can you move this 5 lines below (in the `$modules/database/mysql/mysql_user.py:` section)?